### DISCUSSION TITLE
App price

### DISCUSSION THREAD

**Discussion by areteeem** [inventree/InvenTree#7607] [Q&A]
Hi there! I'm planning on running a small business, and as I'm from Ukraine, many services are blocking our country(_Square, Loyverse, ShipStation and other inventory managing software/POS systems_) from signing up and working stable. So as a small business pre-owner let's say, I can't currently afford the mobile app (_In our country it's 230 hryvnas to be precise, and I know that it's just a couple dollars_), so is there any way to **install it for free using APK**?

**Comment by matmair** [comment]
The app can be built for free, instructions are in the readme https://github.com/inventree/InvenTree-app. Automatic updates and developer certificates cost money and time.
